releasenotes/notes/issue-5751-1b6249f6263c9c30.yaml

```diff
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+    This PR introduces a new transpiler pass capable of generating calibrations
+    and adding them to a quantum circuit. In this PR we introduce the
+    RZXCalibrationBuilder which takes calibrated CNOT gates and creates the
+    calibrations for RZXGates with an arbitrary rotation angle. The schedules
```

Comment:
```suggestion
    Introduced a new transpiler pass capable of generating calibrations
    and adding them to a quantum circuit. The
    :py:class:`~qiskit.transpiler.passes.RZXCalibrationBuilder` takes calibrated
    CNOT gates and creates the
    calibrations for RZXGates with an arbitrary rotation angle. The schedules
```

Reply:
I removed references to the current PR as the release notes will all get aggregated at release time and the PR pointer will be lost